network-store-model/src/main/java/com/powsybl/network/store/model/TapChangerParentAttributes.java

```diff
@@ -16,4 +18,13 @@
 
     void setRatioTapChangerAttributes(RatioTapChangerAttributes ratioTapChangerAttributes);
 
+    RatioTapChangerAttributes getRatioTapChangerAttributes();
+
+    PhaseTapChangerAttributes getPhaseTapChangerAttributes();
+
+    Set<TapChangerAttributes> getAllTapChangersAttributes();
```

Comment:
getAllTapChangersAttributes, hasRatioTapChangerAttributes, hasPhaseTapChangerAttributes have to be removed from this class. This a data object, automatically serialized by jackson and adding this kind of method is dangerous. I suspect this methods to be used in jackson serialization.
Adding ignored annotation is also not a clean solution. We should move utility methods outside of resources.

Reply:
done